docs/whatsnew/0.9.rst

```diff
@@ -0,0 +1,152 @@
+.. doctest-skip-all
+
+.. _whatsnew-0.9:
+
+************************
+What's New in SunPy 0.9?
+************************
+
+Overview
+========
+
+SunPy 0.9 brings improved support for downloading data from the JSOC
+and bugfixes compared to the 0.8.x series of releases.  The 0.9.x
+series will be the last series of SunPy releases to support Python 2.
+This is because Python 2 will not be maintained after 2019 - see
+https://pythonclock.org/.  The 0.9.x series will receive bugfix
```

Comment:
I think it would be better to link to this: https://python3statement.org/

(I am updating our releases on it now)